### DISCUSSION TITLE
Help using the GPT-4-vision-preview model

### DISCUSSION THREAD

**Discussion by saatchi-david** [FlowiseAI/Flowise#1402] [Q&A]
Hi, I’m looking for help with how to use the gpt4-vision-preview model. 

I’ve tried setting it as the LM for a conversational agent and passing it a url but I get a JSON error (regardless of what I prompt). 

Has anyone had any success? Appreciate any help!

**Comment by delinecnlin** [comment]
It would be nice if the vision mode also supports Azure OpenAI
